app/assets/javascripts/3dbio_viewer/src/webapp/components/molecular-structure/usePluginRef.ts

```diff
@@ -177,11 +276,87 @@ export function usePluginRef(options: Options) {
                     .catch(err => loadVoidMolstar(err));
             }
 
-            if (pluginAlreadyRendered) {
+            function setVisibilityAfterLoad(initParams: InitParams, plugin: PDBeMolstarPlugin) {
+                molstarState.current = MolstarStateActions.fromInitParams(initParams, newSelection);
+
+                if (newSelection.type === "free") {
+                    if (newSelection.main.pdb) setVisibility(plugin, newSelection.main.pdb);
+                    if (newSelection.main.emdb) {
+                        setVisibility(plugin, newSelection.main.emdb);
+                        setEmdbOpacity({ plugin, id: newSelection.main.emdb.id, value: 0.5 });
+                    }
```

Comment:
the layout of conditional blocks is very important for its readability, let's use also block style for the `if (newSelection.main.pdb)` . Homogeneous code is easier to read.